src/components/Drawer/Drawer.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import CartItem from "./CartItem/CartItem";
+import styles from "./Drawer.module.scss";
+
+type DrawerProps = {
+    onClickCart: () => void;
+}
```

Comment:
`import Item type from shop`

```suggestion
    items: Item[];
}
```

Reply:
fixed